Dockerfile

```diff
@@ -1,72 +1,68 @@
-FROM python:3.9-slim
+# We use Debian images because they are considered more stable than the alpine
+# ones becase they use a different C compiler. Debian images also come with
+# all useful packages required for image manipulation out of the box. They
+# however weigh a lot, approx. up to 1.5GiB per built image.
+FROM python:3.9 as production
```

Comment:
my main gripe with this is that the image is quite a bit bigger now...

Reply:
Yeah it gets a **lot** bigger. There are some ways we can reduce the size of the image I've been meaning to try. Perhaps this is the time to try them... :thinking:

Reply:
I've made some changes internally to our image configuration. If they get reviewed and approved, we can update things here too. 45% improvement is definitely worthwhile.

Reply:
Do we gain anything by switching to the full-size image here? Appreciate we are willing to live with the tradeoffs in Torchbox projects, but as a recommended approach for beginners, is slim not preferrable?